packages/emails/index.ts

```diff
@@ -1,12 +1,19 @@
 import type { ReactElement } from "react";
 import { Resend } from "resend";
 
+import Alert, { EmailDataSchema } from "./emails/alert";
```

Comment:
Let's call it `AlertEmailDataSchema` as we will have multiple `EmailDataSchema`s in the future.